std/outbuffer.d

```diff
@@ -269,21 +272,37 @@ class OutBuffer
         {
             version(Windows)
             {
-                count = vsnprintf(p,psize,f,args);
+                va_list args2;
+                va_copy(args2, args);
+                count = vsnprintf(p,psize,f,args2);
+                va_end(args2);
                 if (count != -1)
                     break;
-                psize *= 2;
+
```

Comment:
Another overkill change. If you really insist:

`psize <= psize.max / 2 || assert(0);`